packages/webpack-plugin/src/stylable-webpack-plugin.ts

```diff
@@ -177,6 +151,32 @@ export class StylableWebpackPlugin {
         });
         this.injectStylableCSSOptimizer(compiler);
     }
+    private processStylableModuleAssets(
+        compilation: webpack.compilation.Compilation,
+        module: StylableModule
+    ) {
+        const rootContext = (compilation as any).options.context;
+        const replacements: WebpackAssetModule[] = [];
+        const moduleDir = dirname(module.resource);
+        const onUrl = (node: any) => {
```

Comment:
`const onUrl: OnUrlCallback = (node) => {`
which will then show you we need to protect against `node.url` being `undefined` before calling isAsset